request-handler.js

```diff
@@ -54,8 +54,19 @@ RequestCallbackHandler.prototype.handleRequest = function handleRequest(req, bui
     var res;
     if (req.streamed) {
         req.withArg23(function onArg23(err, arg2, arg3) {
+            if (err) {
+                req.channel.logger.warn(
+                    'Could not parse arg3 for streaming inreq',
+                    req.extendLogInfo({
+                        error: err
+                    })
+                );
+
+                return req.emitError(err);
```

Comment:
won't it log about it? shouldn't we just wrap it here in a `arg23somethingerorr'? maybe `req#withArg23` should do such wrapping for us?